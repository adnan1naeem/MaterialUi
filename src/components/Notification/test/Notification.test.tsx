// import React from 'react';
// import Adapter from 'enzyme-adapter-react-16';
// import renderer from 'react-test-renderer';
// import { Notification } from '../';
// import { configure, mount, shallow, } from 'enzyme';
// configure({ adapter: new Adapter() });
// test('render correctly', () => {
//   const notification = shallow(
//     <Notification
//       open={() => console.log('open')}
//       onClose={() => console.log('open')}
//       transition={''}
//       SnackbarContentProps={{ 'aria-describedby': 'message-id' }}
//       text={<span id="message-id">I love snacks</span>}
//     />
//   );
//   expect(notification).toBeDefined();
//   expect(notification).toMatchSnapshot();
// });