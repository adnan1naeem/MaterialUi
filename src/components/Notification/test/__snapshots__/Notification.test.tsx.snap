// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    SnackbarContentProps={
        Object {
            "aria-describedby": "message-id",
          }
    }
    onClose={[Function]}
    open={[Function]}
    text={
        <span
            id="message-id"
        >
            I love snacks
        </span>
    }
    transition=""
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "SnackbarContentProps": Object {
        "aria-describedby": "message-id",
      },
      "autoHideDuration": undefined,
      "message": <span
        id="message-id"
>
        I love snacks
</span>,
      "onClose": [Function],
      "open": [Function],
      "text": <span
        id="message-id"
>
        I love snacks
</span>,
      "transition": "",
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "SnackbarContentProps": Object {
          "aria-describedby": "message-id",
        },
        "autoHideDuration": undefined,
        "message": <span
          id="message-id"
>
          I love snacks
</span>,
        "onClose": [Function],
        "open": [Function],
        "text": <span
          id="message-id"
>
          I love snacks
</span>,
        "transition": "",
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
